doc/ref/configuration/master.rst

```diff
@@ -1925,6 +1925,42 @@ the cloud profile or master config file, no templating will be performed.
 
     userdata_template: jinja
 
+.. conf_master:: jinja_line_statement_prefix
+
+``jinja_line_statement_prefix``
+---------------------
+
+.. versionadded:: Oxygen
+
+Default: ``''``
+
+If this is set to a string that is not empty, any line starting with this prefix
+will be interpreted as a line statement by the jinja renderer.
+Defaults to ``''`` and corresponds
```

Comment:
``''`` dont corespond with and commented variant above "#"
also I don't find ``''`` anyway usefull thant "#". Instead of I would suggest `%` as the char. is there anything agains `%` or ``%%`` ? I think it would be more close to today practice.

Reply:
Most of the documentation (including this file) is outdated with the code and I will likely have to rewrite most of it. The `#` and `##` suggestions were just pulled from the example in the jinja documentation but I agree that `%` and `%%` would be much better for sls files. The only conflict I see is that `%` is used for [yaml directives](http://yaml.org/spec/1.2/spec.html#id2806177) but I don't think they are used often (if at all) in salt's sls files. Will change the suggestions to use `%` and `%%`.